examples_extra/ASDKgram-Swift/ASDKgram-Swift/PhotoTableNodeCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  PhotoTableNodeCell.swift
+//  ASDKgram-Swift
+//
+//  Created by Calum Harris on 09/01/2017.
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+//  FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+//   ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+//  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.//
+
+import Foundation
+import AsyncDisplayKit
+
+class PhotoTableNodeCell: ASCellNode {
+
+	init(photoModel: PhotoModel) {
+		super.init()
+		self.photoImageNode.url = URL(string: photoModel.url)
+		self.avatarImageNode.url = URL(string: photoModel.ownerPicURL)
+		self.usernameLabel.attributedText = photoModel.attrStringForUserName(withSize: Constants.CellLayout.FontSize)
+		self.timeIntervalLabel.attributedText = photoModel.attrStringForTimeSinceString(withSize: Constants.CellLayout.FontSize)
+		self.photoLikesLabel.attributedText = photoModel.attrStringLikes(withSize: Constants.CellLayout.FontSize)
+		self.photoDescriptionLabel.attributedText = photoModel.attrStringForDescription(withSize: Constants.CellLayout.FontSize)
+		self.automaticallyManagesSubnodes = true
+	}
+
+	let photoImageNode: ASNetworkImageNode = {
+		let imageNode = ASNetworkImageNode()
+		imageNode.contentMode = .scaleAspectFill
+		return imageNode
+	}()
+
+	var avatarImageNode: ASNetworkImageNode = {
+		let imageNode = ASNetworkImageNode()
+		imageNode.contentMode = .scaleAspectFill
+		imageNode.imageModificationBlock = ASImageNodeRoundBorderModificationBlock(0, nil)
+		return imageNode
+	}()
+
+	let usernameLabel: ASTextNode = {
+		let label = ASTextNode()
+		return label
+	}()
+
+	let timeIntervalLabel: ASTextNode = {
+		let label = ASTextNode()
+		return label
+	}()
+
+	let photoLikesLabel: ASTextNode = {
+		let label = ASTextNode()
+		return label
+	}()
+
+	let photoDescriptionLabel: ASTextNode = {
+		let label = ASTextNode()
+		return label
+	}()
```

Comment:
Let's move all the property declarations above `init` and trim these down to just

```swift
let usernameLabel = ASTextNode()
let timeIntervalLabel = ASTextNode()
etc.
```